### DISCUSSION TITLE
Skin Collection/Folders

### DISCUSSION THREAD

**Discussion by ItsMeImmis** [ppy/osu#27892] [Ideas]
Hi! I just wanted to give an idea for better skin sorting (especially since a regular player may have 50+ skins). I made an edit down below.

![osulazerskinedit](https://github.com/ppy/osu/assets/165425980/52b985a1-89b9-48f1-bdf1-abdc897a2eb5)

I figured this quality of life feature is great even more so when some skins have a vague name or a user probably forgot the name.  The numbers could be limited to skin names that match the search box. 

I feel like this can also support a shortcut (Crtl + Shift + , or . ) where you can cycle to previous or next skin in a collection. The randomize skin shortcut can also be limited to only skins within a collection.

**Comment by mixalisnew** [comment]
yes pls

**Reply by EWReinE** [reply]
I agree, this is something very much needed in lazer. Although you could rename all the skins with "(x)skin", I find that very tedious (speaking from experience), so the implementation of a folder feature like above would be greatly welcomed.

**Comment by PinkDucki** [comment]
retweet

**Comment by Xeladoes** [comment]
pls !

**Comment by kyiuu1** [comment]
pls!!

**Comment by I-AmBen** [comment]
I want this.
